lib/templ.ml

```diff
@@ -436,11 +541,12 @@ let rec eval_expr (conf, eval_var, eval_apply as ceva) =
       | "!=" -> VVbool (eval_expr ceva e1 <> eval_expr ceva e2)
       | "<=" -> VVbool (int e1 <= int e2)
       | ">=" -> VVbool (int e1 >= int e2)
-      | "+" -> VVstring (Sosa.to_string (Sosa.add (num e1) (num e2)))
-      | "-" -> VVstring (Sosa.to_string (Sosa.sub (num e1) (num e2)))
-      | "*" -> VVstring (Sosa.to_string (Sosa.mul (num e1) (int e2)))
+      | "+" -> VVstring (string_of_int ((int e1) + (int e2)))
+      | "-" -> VVstring (string_of_int ((int e1) - (int e2)))
+      | "*" -> VVstring (string_of_int ((int e1) * (int e2)))
       | "^" -> VVstring (Sosa.to_string (Sosa.exp (num e1) (int e2)))
       | "/" -> VVstring (Sosa.to_string (Sosa.div (num e1) (int e2)))
+      | "/." -> VVstring (strip_dot (Float.to_string (Float.div (float e1) (float e2))))
```

Comment:
maybe you want to use Float.is_integer and Float.to_int instead of using strip_dot on the string

Reply:
ok, I will look